pkg/api/types.go

```diff
@@ -1041,6 +1041,95 @@ const (
 	DNSDefault DNSPolicy = "Default"
 )
 
+// A node selector represents the union of the results of one or more label queries
+// over a set of nodes; that is, it represents the OR of the selectors represented
+// by the nodeSelectorTerms.
```

Comment:
s/nodeSelectorTerms/node selector terms/
also, mention that a null NodeSelector matches all objects, i.e. does not restrict scheduling